library/Scientist/Duration.hs

```diff
@@ -1,21 +1,54 @@
 {-# LANGUAGE DerivingStrategies #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+
+
 
 module Scientist.Duration
-  ( Duration(..)
+  ( Duration
   , measureDuration
+  , toNanoSecs
+  , fromNanoSecs
   ) where
 
 import Prelude
 
 import Control.Monad.IO.Class (MonadIO(..))
-import Data.Time (NominalDiffTime, diffUTCTime, getCurrentTime)
+import Data.Fixed (Fixed(..), Nano, showFixed)
+import qualified System.Clock as Clock
 
+-- | Time elapsed in seconds up to nanosecond precision
+--
+-- >> 1.005
+-- 1.005s
+--
+-- >> toNanoSecs 1.005
+-- 1005000000
```

Comment:
Might as well make then valid doctest, even if we don't set up a runner yet. Also, we put whitespace around documentation bodies when present so it feels less cramped.

```suggestion
-- >>> 1.005
-- 1.005s
--
-- >>> toNanoSecs 1.005
-- 1005000000
--
```